OpenChart/src/Charting/ChartUtils.cs

```diff
@@ -10,19 +10,15 @@ public static class ChartUtils
         /// slightly inaccurate.
         /// </summary>
         /// <param name="bpms">The BPM changes. Cannot be empty.</param>
-        /// <param name="beat">The beat number. Must be greater than zero.</param>
+        /// <param name="_beat">The beat number. Must be greater than zero.</param>
```

Comment:
this param name does not match the actual parameter name

Reply:
Always use the refactor function instead of find and replace, boys and girls